packages/schema/src/types/hooks.ts

```diff
@@ -28,6 +28,7 @@ export type NuxtPage = {
   alias?: string[] | string
   redirect?: RouteLocationRaw
   children?: NuxtPage[]
+  mode?: 'server'
```

Comment:
Could we support `all` as well, matching component mode syntax?

```suggestion
  mode?: 'server' | 'all'
```